### PR TITLE
feat(182215592): users trip request

### PR DESCRIPTION
**Pull request by IgihozoColombe**
**What does this PR do?**

Trip Request CRUD.The Trip request can be created,edited and deleted by user 
and also the admin can view all trip requests

**Description of Task to be completed?**
Endpoint POST `/api/v1/trips`
Endpoint GET `/api/v1/trips`
Endpoint GET` /api/v1/trips/:id`
Endpoint PUT `/api/v1/trips/:id`
Endpoint DELETE `/api/v1/trips/:id`

**How should this be manually tested?**
clone project and` run npm i`
test endpoint Endpoints POST /api/v1/trips
                       GET /api/v1/trips
                       GET /api/v1/trips/:id
                       PUT /api/v1/trips/:id
                       DELETE /api/v1/trips/:id
**What are the relevant pivotal tracker stories?**
#182215592

### CODE REVIEW

**Review by IsiteYves — APPROVED** [review]

**Review by IsiteYves — CHANGES_REQUESTED** [review]
Try to use the correct API naming convention.For Endpoint GET `/api/v1/trips/all`, it should  have been GET `/api/v1/trips` only.

**Review by IsiteYves — COMMENTED** [review]
Try to use the suggested API naming convention.For endpoint GET `/api/v1/trips/all`, it should have been GET `/api/v1/trips` only

**Review by NezaBruce — APPROVED** [review]

**Review by kelliaUmuhire — APPROVED** [review]

**Review by IsiteYves — APPROVED** [review]

**Review by leny62 — APPROVED** [review]

**Comment by leny62** [comment]
@IgihozoColombe , check the circle-ci build and test issue

**Comment by leny62** [comment]
@IgihozoColombe , check the circle-ci build and test issue

**Comment by leny62** [comment]
@IgihozoColombe , check the circle-ci build and test issue

**Review by kelliaUmuhire — APPROVED** [review]

**Review by pmuhire — APPROVED** [review]

File: models/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
```
Comment (leny62) [inline models/index.js:4]: Please use ES6

File: src/models/index.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
-'use strict';
-
 const fs = require('fs');
 const path = require('path');
 const Sequelize = require('sequelize');
```
Comment (leny62) [inline src/models/index.js:3]: Please use ES6

File: src/models/triprequest.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+const {
```
Comment (leny62) [inline src/models/triprequest.js:2]: Please use ES6

**Review by leny62 — CHANGES_REQUESTED** [review]
Good progress @IgihozoColombe , comply to the requested changes of changing to ES6 and also Rebase!

**Pull request closed without merging**